### PR TITLE
Extract Hello World Blog Post

### PR DESCRIPTION
**Pull request by MarkBennett**
I've pulled your post into it's own PR @johnny-newton.

As part of this I also re-sized the image to fit better on the web as it was too big, and dropped the size of the image attribution to look more like a caption.

### CODE REVIEW

File: _posts/2020-04-05-hello.md
Comment (MarkBennett) [inline _posts/2020-04-05-hello.md:9]: Not supporting Markdown in HTML is the one part of Markdown spec that drives me crazy. This looks so ugly, but 🤷

**Comment by johnnewton-hub** [comment]
Thanks Mark!

I’m going back into the comments on git, to see what’s all been done.  Aaron just spent a good hour walking me through github mojo..and I’m definitely knowledge up-levelled.

jn

File: _posts/2020-04-05-hello.md
Comment (ajyong) [reply _posts/2020-04-05-hello.md:9]: I'm not going to block this PR, but I'm strongly against using HTML, even if it leads to richer layouts like this.

1. It breaks the blog list page's excerpt capability, which blocks the ability for me to add a link to our blog from the home page.
   image
1. Subjectively, the image and its attribution don't respond well to mobile
   image

It's not the end of the world, but I think I'll submit a mini-PR after the fact to just make this easier by making it an image with some of the first and bottom third of it cropped out.

**Review by ajyong — APPROVED** [review]
LGTM with a nitpick that can be addressed in a separate PR.

**Comment by ajyong** [comment]
I guess since this is a Franken-PR where @johnny-newton made the original contributions but @MarkBennett cleaned it up, either of you can merge it.  Conventionally though, it's been the PR author who merges after approval.

**Comment by MarkBennett** [comment]
Thanks @ajyong! Merged!

**Pull request merged**
